src/main/java/com/bamdoliro/maru/domain/question/domain/Question.java

```diff
@@ -0,0 +1,42 @@
+package com.bamdoliro.maru.domain.question.domain;
+
+import com.bamdoliro.maru.presentation.question.dto.request.UpdateQuestionRequest;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Table(name = "tbl_question")
+@Entity
+public class Question {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "question_id")
+    private Long id;
```

Comment:
```suggestion
    @Column(name = "question_id")
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Id
    private Long id;
```
연관성 있는 어노테이션이 가까이 붙게 해주세여